### PR TITLE
changed osbb views and account layout view

### PR DESCRIPTION
**Pull request by matsevich**
[Trello Task](https://trello.com/c/yPQeL0sT/62-fixing-account-layout-view-and-osbb-views)

**CHECK LIST**
- [ ] specs added
- [ ] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

Screenshot from 2020-10-22 17-07-12
Screenshot from 2020-10-22 17-07-18
Screenshot from 2020-10-22 17-07-24

### CODE REVIEW

**Review by Ukhanskyi — APPROVED** [review]

**Review by DenisRem — APPROVED** [review]

File: app/controllers/account/admin/billing_contracts_controller.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def destroy
   private
 
   def billing_contract_params
-    params.require(:billing_contract).permit(:contract_num, :is_active)
+    params.require(:billing_contract).permit(:contract_num, :is_active, :user_id, :company_id)
```
Comment (chu4) [inline app/controllers/account/admin/billing_contracts_controller.rb:50]: what for is user_id?

**Review by Axedez — APPROVED** [review]

**Comment by chu4** [comment]
and forms look very wide, would more visually appealing if they were more narrow

**Review by marikrebega — APPROVED** [review]

**Review by Andrii-Kozak — APPROVED** [review]

File: app/controllers/account/admin/billing_contracts_controller.rb (hunk 1 above)
Comment (matsevich) [reply app/controllers/account/admin/billing_contracts_controller.rb:50]: they are in the table of billing_contracts, so later we will need them here. I may delete them if you want...

Comment (vasya539) [reply app/controllers/account/admin/billing_contracts_controller.rb:50]: > they are in the table of billing_contracts, so later we will need them here. I may delete them if you want...

don't add those "later" things, it's better to add them at the time they should be used

**Review by chu4 — APPROVED** [review]

**Pull request merged**